bgpd/bgp_route.c

```diff
@@ -4182,6 +4182,12 @@ static void bgp_process_internal(struct bgp *bgp, struct bgp_dest *dest,
 	return;
 }
 
+void bgp_process_hook(struct bgp *bgp, struct peer *peer, struct bgp_dest *dest, afi_t afi,
```

Comment:
ok, I'll bite: why?
why add a layer that just rearranges the arguments, and hides the "withdraw" boolean?

Reply:
- the layer, it is because it is not possible to call a HOOK from outside of bgp_route.c
- the withdraw, I can maintain it.

Reply:
OK, even if parameters are similar, this is not exactly the correct hook I wanted to call.
BMP is interested in having nexthop tracking hooks, in order to consider a path as reachable or not, and then further update the post-rib.